DEVELOPMENT.md

```diff
@@ -275,48 +275,66 @@ Kubernetes service calls get redirected to the process on your local. Similarly
 the calls on the local process goes to actual services that are running in
 Kubernetes.
 
-- Install Telepresence
+### Prerequisites
 
+- Install Telepresence v2 (see the [installation instructions](https://www.getambassador.io/docs/telepresence/latest/install/) for details).
 - Deploy Knative Eventing on your Kubernetes cluster.
 
-- Install [EnvFile plugin](https://plugins.jetbrains.com/plugin/7861-envfile) to
-  your Intellij
+### Connect Telepresence and intercept the controller
 
-- Run following command to swap the controller with the controller that we will
-  start later.
+As a first step Telepresence needs to your Kubernetes cluster:
 
 ```
-telepresence --namespace knative-eventing --swap-deployment eventing-controller --env-json eventing-controller-local-env.json
+telepresence connect
 ```
 
-For debugging applications that receive traffic, such as webhooks, you also need
-to pass `--expose` parameter.
+_Hint: If this is your first time Telepresence connects to your cluster, you
```

Comment:
I'd add this hint _before_ the `telepresence connect` (since connect otherwise fails)